frontend/appflowy_flutter/lib/workspace/presentation/home/menu/app/header/header.dart

```diff
@@ -86,6 +88,9 @@ class MenuAppHeader extends StatelessWidget {
                 value: context.read<AppBloc>().state.app.name,
                 confirm: (newValue) {
                   context.read<AppBloc>().add(AppEvent.rename(newValue));
+                  context
+                      .read<DocumentBloc>()
```

Comment:
The `DocumentBloc` isn't provided in the current context. Are you sure that you can access it here?